models/istio_validation.go

```diff
@@ -303,7 +303,7 @@ var checkDescriptors = map[string]IstioCheck{
 	},
 	"virtualservices.route.repeatedsubset": {
 		Code:     "KIA1105",
-		Message:  "This subset is already referenced in another route destination",
+		Message:  "This host subset combination is already referenced in another route destination",
```

Comment:
Minor rewording, "host subset" -> "host/subset" ?

Reply:
it is used 'host subset' also in "destinationrules.multimatch", might be changing both?

Reply:
Nah, it's very minor, I'm also fine with the wording and if it's used in other places I don't see a I need to change.